pkg/apis/app/addtoscheme_v1alpha1.go

```diff
@@ -5,10 +5,10 @@
 package apis
 
 import (
-	"github.com/lyft/flinkk8soperator/pkg/apis/app/v1alpha1"
+	"github.com/lyft/flinkk8soperator/pkg/apis/app/v1beta1"
 )
 
 func init() {
 	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
-	AddToSchemes = append(AddToSchemes, v1alpha1.SchemeBuilder.AddToScheme)
+	AddToSchemes = append(AddToSchemes, v1beta1.SchemeBuilder.AddToScheme)
```

Comment:
The file name indicates `addtoscheme_v1alpha1.go`

Reply:
fixed